src/extensions/extensions-store.ts

```diff
@@ -34,11 +34,6 @@ export class ExtensionsStore extends BaseStore<LensExtensionsStoreModel> {
     await extensionLoader.whenLoaded;
     await this.whenLoaded;
 
-    // activate user-extensions when state is ready
```

Comment:
extensionManager has not loaded extensions at this point so we cannot determine the status here.